src/App.css

```diff
@@ -22,18 +22,6 @@
   visibility: visible;
 }
 
-.usa-alert button {
```

Comment:
I love how it took me like 25 minutes of scrolling to find out what happened to this code. 😁 

Though actually most of it is still dead—were we just no longer using it? I assume we didn't just add it for fun (well, probably not...)